src/components/study-pane.js

```diff
@@ -702,8 +772,18 @@ const StudyPane = (props) => {
         history.replace('/studies/' + encodeURIComponent(studyUuid));
     }
 
-    const toggleDrawer = () => {
-        setDrawerOpen(!drawerOpen);
+    const toggleExplorerDrawer = () => {
+        setDrawerExplorerOpen(!drawerExplorerOpen);
+        if (drawerHypoTreeOpen) {
+            setDrawerHypoTreeOpen(!drawerHypoTreeOpen);
```

Comment:
May be, explicit false, in stead of using the negator of positive/true value just evaluated ?

Reply:
Yes, I removed if (drawerHypoTreeOpen) condition and set value explicitely to false.